xarray/plot/facetgrid.py

```diff
@@ -221,6 +221,14 @@ def map_dataarray(self, func, x, y, **kwargs):
         self : FacetGrid object
 
         """
+
+        cmapkw = kwargs['cmap'] if 'cmap' in kwargs else None
+        colorskw = kwargs['colors'] if 'colors' in kwargs else None
```

Comment:
Use `get` here, e.g., `kwargs.get('cmap')`.

Reply:
Nice, that's a great feature.